typings/sublime_plugin.pyi

```diff
@@ -588,7 +588,7 @@ class CommandInputHandler(Generic[T_InputType]):
         """
         ...
 
-    def next_input(self, args: Dict[str, Any]) -> None | CommandInputHandler[T_InputType]:
+    def next_input(self, args: Dict[str, Any]) -> None | CommandInputHandler[Any]:
```

Comment:
I still wonder about this. `Any` would violate the restrictions of the typvar in `CommandInputHandler`.

We have two options.

1. Add a second typevar that is bound to `None | CommandInputHandler` and have the user handle that.
2. Define a `Value` type alias and then use that:
```py
Value = None | str | int | float | Dict[Any, Any] | List[Any] | Tuple[Any, ...]
T_InputType = TypeVar("T_InputType", bound=Value)
```

```suggestion
    def next_input(self, args: Dict[str, Any]) -> None | CommandInputHandler[Value]:
```

To make a proper decision, I'd have to play around with the stubs with tools and some actual code.